### PR TITLE
readme:chore - Added new fixes to readme

### PR DESCRIPTION
**Pull request by jessica-tw**
Signed-off-by: jessica-tw <jessica.olinda@zup.com.br>

**- What I did**
- Fixed some issues and reviewed README.md and Contributing.md one more time. 

**- How to verify it**

**- Description for the changelog** Added new fixes to Readme and Contributing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,29 +26,31 @@
...
-        5. [**Using Visual Studio Code**](#Using-Visual-Studio-Code)
-        6. [**Using the Pipeline**](#Using-the-Pipeline)
-4. [**Contributing**](#Contributing)
-5. [**Roadmap**](#Community)
-6. [**License**](#License)
-7. [**Community**](#Community)
+### 1. [**About**](#About)
+### 2. [**Getting started**](#Getting-started)
+>#### i.   [**Requirements**](#Requirements)
+>#### ii.  [**Installation**](#Installation)
```
Comment (wiliansilvazup) [inline README.md:32]: link of redirect is wrong, the correctly is 
```suggestion
>#### ii.  [**Installation**](#installing-horusec)
```

File: README.md [hunk 2]
```diff
@@ -26,29 +26,31 @@
...
+>#### iii. [**Usage**](#Usage)
```
Comment (wiliansilvazup) [inline README.md:33]: link of redirect is wrong, the correctly is 
```suggestion
>#### iii. [**Usage**](#usage)
```

File: README.md [hunk 3]
```diff
@@ -26,29 +26,31 @@
...
+>##### [**CLI Usage**](#CLI-Usage)
```
Comment (wiliansilvazup) [inline README.md:34]: link of redirect is wrong, the correctly is
```suggestion
>##### [**CLI Usage**](#cli-usage-1)
```

File: README.md [hunk 4]
```diff
@@ -26,29 +26,31 @@
...
+>##### [**Using Docker**](#Using-Docker)
```
Comment (wiliansilvazup) [inline README.md:35]: link of redirect is wrong, the correctly is
```suggestion
>##### [**Using Docker**](#using-docker-1)
```

File: README.md [hunk 5]
```diff
@@ -26,29 +26,31 @@
...
+>##### [**Older versions**](#Older-versions)
```
Comment (wiliansilvazup) [inline README.md:36]: link of redirect is wrong, the correctly is

```suggestion
>##### [**Older versions**](#older-versions-1)
```

File: README.md [hunk 6]
```diff
@@ -26,29 +26,31 @@
...
+>##### [**Using Horusec-Web application**](#Using-Horusec-Web-application)
```
Comment (wiliansilvazup) [inline README.md:37]: link of redirect is wrong, the correctly is
```suggestion
>##### [**Using Horusec-Web application**](#using-horusec-web-application-1)
```

File: README.md [hunk 7]
```diff
@@ -26,29 +26,31 @@
...
+>##### [**Using Visual Studio Code**](#Using-Visual-Studio-Code)
```
Comment (wiliansilvazup) [inline README.md:38]: link of redirect is wrong, the correctly is

```suggestion
>##### [**Using Visual Studio Code**](#using-visual-studio-code-1)
```

File: README.md [hunk 8]
```diff
@@ -26,29 +26,31 @@
...
+>##### [**Using the Pipeline**](#Using-the-Pipeline)
```
Comment (wiliansilvazup) [inline README.md:39]: link of redirect is wrong, the correctly is

```suggestion
>##### [**Using the Pipeline**](#using-visual-studio-code-1)
```

Comment (wiliansilvazup) [reply README.md:39]: You can test in: https://github.com/ZupIT/horusec/tree/fix/readme

File: README.md (hunk 7 above)
Comment (wiliansilvazup) [reply README.md:38]: You can test in: https://github.com/ZupIT/horusec/tree/fix/readme

File: README.md (hunk 6 above)
Comment (wiliansilvazup) [reply README.md:37]: You can test in: https://github.com/ZupIT/horusec/tree/fix/readme

File: README.md (hunk 5 above)
Comment (wiliansilvazup) [reply README.md:36]: You can test in: https://github.com/ZupIT/horusec/tree/fix/readme

File: README.md (hunk 4 above)
Comment (wiliansilvazup) [reply README.md:35]: You can test in: https://github.com/ZupIT/horusec/tree/fix/readme

File: README.md (hunk 3 above)
Comment (wiliansilvazup) [reply README.md:34]: You can test in: https://github.com/ZupIT/horusec/tree/fix/readme

File: README.md (hunk 2 above)
Comment (wiliansilvazup) [reply README.md:33]: You can test in: https://github.com/ZupIT/horusec/tree/fix/readme

File: README.md (hunk 1 above)
Comment (wiliansilvazup) [reply README.md:32]: You can test in: https://github.com/ZupIT/horusec/tree/fix/readme

File: README.md [hunk 9]
```diff
@@ -168,6 +170,20 @@ See below:
...
+## **Roadmap**
+We have a project [**roadmap**](ROADMAP.md), you can come contribute with us!
+
+Horusec also have other repositories, check out some of our other projects:
+
+- [**Horusec Platform**](https://github.com/ZupIT/horusec-platform)
+- [**Horusec DevKit**](https://github.com/ZupIT/horusec-devkit)
+- [**Horusec Engine**](https://github.com/ZupIT/horusec-engine)
+- [**Horusec Operator**](https://github.com/ZupIT/horusec-operator)
+- [**Horusec Admin**](https://github.com/ZupIT/horusec-admin)
```
Comment (nathanmartinszup) [inline README.md:185]: Since admin is deprecated, should we keep it? 

@wiliansilvazup

Comment (wiliansilvazup) [reply README.md:185]: We can remove 
```suggestion
```

**Review by wiliansilvazup — CHANGES_REQUESTED** [review]
I'll wait fix this changes :)

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -3,31 +3,28 @@
...
-      2. [**First contribution**](#First-contribution)
-      4. [**Add new feature, bugfixing or improvement**](#Add-new-feature-bugfixing-or-improvement)
-      5. [**Pull Request's approval**](#Pull-Request's-approval)
-      6. [**After your pull request's approval**](#After-your-pull-request's-approval)
-4. [**Community**](#Community)
-
-## **Getting started**
-
-### **Before you contribute**
+### 1. [**Before you contribute**](#Before-you-contribute)
```
Comment (nathanmartinszup) [inline CONTRIBUTING.md:6]: The following buttons are not redirecting to the content:

[**Developer Certificate of Origin**](#Developer-Certificate-of-Origin)
[**Pull Request's approval**](#Pull-Request's-approval)
[**After your pull request's approval**](#After-your-pull-request's-approval)

File: README.md (hunk 1 above)
Comment (nathanmartinszup) [inline README.md:32]: Should the usage assignment follow the numbering?

Example:

## **Table of contents**
### 1. [**About**](#About)
### 2. [**Getting started**](#Getting-started)
>#### i.   [**Requirements**](#Requirements)
>#### ii.  [**Installation**](#Installation)
>#### iii. [**Usage**](#Usage)
>##### i. [**CLI Usage**](#CLI-Usage)
>##### ii. [**Using Docker**](#Using-Docker)
>##### iii. [**Older versions**](#Older-versions)
>##### iv. [**Using Horusec-Web application**](#Using-Horusec-Web-application)
>##### v. [**Using Visual Studio Code**](#Using-Visual-Studio-Code)
>##### vi. [**Using the Pipeline**](#Using-the-Pipeline)
### 3. [**Documentation**](#Documentation)       
### 4. [**Roadmap**](#Roadmap)
### 5. [**Contributing**](#Contributing)
### 6. [**Code of Conduct**](#Code-of-Conduct)
### 7. [**License**](#License)
### 8. [**Community**](#Community)

Comment (jessica-tw) [reply README.md:32]: I did that, but the link wasn't working, so I removed it. I will try again.

File: README.md [hunk 11]
```diff
@@ -26,29 +26,31 @@
...
+### 2. [**Getting started**](#getting-started)
+>#### i.   [**Requirements**](#requirements)
+>#### ii.  [**Installation**](#installing-horusec)
+>#### iii. [**Usage**](#usage)
+>##### i. [**CLI Usage**](#cli-usage-1)
+>##### ii. [**Using Docker**](#using-docker-1)
+>##### iii. [**Older versions**](#older-versions-1)
+>##### iv. [**Using Horusec-Web application**](#using-horusec-web-application-1)
+>##### v.  [**Using Visual Studio Code**](#using-visual-studio-code-1)
+>##### vi. [**Using the Pipeline**](#Using-the-Pipeline)
```
Comment (wiliansilvazup) [inline README.md:39]:
```suggestion
> #### 2.1. [**Requirements**](#requirements)
> #### 2.2. [**Installation**](#installing-horusec)
> #### 2.3. [**Usage**](#usage)
>> ##### 2.3.1. [**CLI Usage**](#cli-usage-1)
>> ##### 2.3.2. [**Using Docker**](#using-docker-1)
>> ##### 2.3.3. [**Older versions**](#older-versions-1)
>> ##### 2.3.4. [**Using Horusec-Web application**](#using-horusec-web-application-1)
>> ##### 2.3.5. [**Using Visual Studio Code**](#using-visual-studio-code-1)
>> ##### 2.3.6. [**Using the Pipeline**](#Using-the-Pipeline)
```

Roman numbers can get confusing here wouldn't it be better to stick to the standard numbering?

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -3,31 +3,28 @@
...
+> #### i.   [**Developer Certificate of Origin**](#developer-certificate-of-origin)
+> #### ii.  [**Code Review**](#code-review)
+> #### iii. [**Pull Requests**](#pull-requests)    
+### 3. [**How to contribute?**](#how-to-contribute?)
+ > #### i. [**Prepare your development environment**](#prepare-your-development-environment)
+> #### ii.  [**First contribution**](#first-contribution)
+> #### iii. [**Add new feature, bugfixing or improvement**](#add-new-feature-bugfixing-or-improvement)
+> #### iv.  [**Pull Request's approval**](#pull-request-approval)
+> #### v.   [**After your pull request's approval**](#after-your-pull-request-approval)
+### 4. [**Community**](#community)
```
Comment (wiliansilvazup) [inline CONTRIBUTING.md:19]:
```suggestion
### 1. [**Before you contribute**](#before-you-contribute)
> #### 1.1.  [**Code of Conduct**](#code-of-conduct)
> #### 1.2. [**Legal**](#legal)
### 2. [**Prerequisites**](#prerequisites)
> #### 2.1.   [**Developer Certificate of Origin**](#developer-certificate-of-origin)
> #### 2.2.  [**Code Review**](#code-review)
> #### 2.3. [**Pull Requests**](#pull-requests)    
### 3. [**How to contribute?**](#how-to-contribute?)
> #### 3.1. [**Prepare your development environment**](#prepare-your-development-environment)
> #### 3.2.  [**First contribution**](#first-contribution)
> #### 3.3. [**Add new feature, bugfixing or improvement**](#add-new-feature-bugfixing-or-improvement)
> #### 3.4.  [**Pull Request's approval**](#pull-request-approval)
> #### 3.5.   [**After your pull request's approval**](#after-your-pull-request-approval)
### 4. [**Community**](#community)
```

Roman numbers can get confusing here wouldn't it be better to stick to the standard numbering?

**Review by wiliansilvazup — CHANGES_REQUESTED** [review]
i'll wait you check this changes in table of contents of the README and CONTRIBUTING

File: README.md (hunk 11 above)
Comment (jessica-tw) [reply README.md:39]: I prefer it as well, however it was decided Romans, but we can change! ;)

File: CONTRIBUTING.md [hunk 13]
```diff
@@ -39,36 +36,55 @@ Check out the requisites before contributing to Horusec:
...
+
+**Step 1:** When the commit changes box opens, manually type or paste your signature in the comment box, see the example:
+
 ```
-$ git commit -m “My signed commit” Signed-off-by: username <email address>
+Signed-off-by: Name < e-mail address >
 ```
-Note: For this option, your e-mail must be the same in registered in GitHub. 
+
+For this method, your name and e-mail must be the same registered to your GitHub account.
```
Comment (mdcpmoreira) [inline CONTRIBUTING.md:64]:
```suggestion
For this method, your name and e-mail must be the same registered on your GitHub account.
```

File: CONTRIBUTING.md [hunk 14]
```diff
@@ -39,36 +36,55 @@ Check out the requisites before contributing to Horusec:
...
-- Answer the questions about what you did, how to verify it and a description for the changelog, see an example:
+When you open a Pull Request, follow the requirements below:
+
+1. Add a title with the following pattern: 
+
+#### **[PKG][TYPE]: Description**
+
+#### **PKG:** Name of the package or main service you want to change.
+
+#### **TYPE**: Add what you Pull Request (PR) refers to:
```
Comment (mdcpmoreira) [inline CONTRIBUTING.md:78]:
```suggestion
#### **TYPE**: Add what your Pull Request (PR) refers to:
```

File: CONTRIBUTING.md [hunk 15]
```diff
@@ -77,8 +93,8 @@ When you open a PR, follow the requisites below:
 See the guidelines to submit your changes: 
 
 ### **Prepare your development environment**
-To start contributing with Horusec you first need to install [Go](https://golang.org/dl/). The minimal version required to build is 1.17.
-[GNU Make](https://www.gnu.org/software/make/) is also required to development.
+To start contributing with Horusec you first need to install [**Go**](https://golang.org/dl/). The minimal version required to build is 1.17.
```
Comment (mdcpmoreira) [inline CONTRIBUTING.md:96]:
```suggestion
To start contributing with Horusec, you first need to install [**Go**](https://golang.org/dl/). The minimal version required to build is 1.17.
```

File: README.md [hunk 16]
```diff
@@ -168,6 +170,19 @@ See below:
 - Search for their historical git by secrets and other contents exposed;
 - Your analysis can be fully configurable, [**see all CLI available resources**](https://horusec.io/docs/cli/resources/#3-flags).
 
+## **Documentation**
+You can find Horusec's documentation on our [**website**](https://horusec.io/docs/).
+
+## **Roadmap**
+We have a project [**roadmap**](ROADMAP.md), you can come contribute with us!
+
+Horusec also have other repositories, check out some of our other projects:
```
Comment (mdcpmoreira) [inline README.md:179]:
```suggestion
Horusec has also other repositories, check out some of our other projects:
```

File: README.md [hunk 17]
```diff
@@ -179,40 +194,34 @@ Check out our [**contributing guide**](CONTRIBUTING.md) to learn about our devel
...
-$ git commit -m “My signed commit” Signed-off-by: username <email address>
+Signed-off-by: Name < e-mail address >
 ```
-Note: For this option, your e-mail must be the same in registered in GitHub. 
-
-## **Roadmap**
-We have a project [**roadmap**](ROADMAP.md), you can come contribute with us!
 
-Horusec also have other repositories, check out some of our other projects:
+For this method, your name and e-mail must be the same registered to your GitHub account.
```
Comment (mdcpmoreira) [inline README.md:221]:
```suggestion
For this method, your name and e-mail must be the same registered on your GitHub account.
```

**Review by mdcpmoreira — COMMENTED** [review]
TW review done :)

File: CONTRIBUTING.md [hunk 18]
```diff
@@ -3,31 +3,28 @@
...
-## **Getting started**
-
-### **Before you contribute**
+### 1. [**Before you contribute**](#before-you-contribute)
+> #### 1.1.  [**Code of Conduct**](#code-of-conduct)
+> #### 1.2. [**Legal**](#legal)
+### 2. [**Prerequisites**](#prerequisites)
+> #### 1.1.   [**Developer Certificate of Origin**](#developer-certificate-of-origin)
+> #### 1.2.  [**Code Review**](#code-review)
+> #### 1.3. [**Pull Requests**](#pull-requests)
```
Comment (wiliansilvazup) [inline CONTRIBUTING.md:12]:
```suggestion
> #### 2.1.   [**Developer Certificate of Origin**](#developer-certificate-of-origin)
> #### 2.2.  [**Code Review**](#code-review)
> #### 2.3. [**Pull Requests**](#pull-requests)    
```

the correct would be in this format no?

File: CONTRIBUTING.md [hunk 19]
```diff
@@ -3,31 +3,28 @@
...
+### 2. [**Prerequisites**](#prerequisites)
+> #### 1.1.   [**Developer Certificate of Origin**](#developer-certificate-of-origin)
+> #### 1.2.  [**Code Review**](#code-review)
+> #### 1.3. [**Pull Requests**](#pull-requests)    
+### 3. [**How to contribute?**](#how-to-contribute?)
+ > #### 1.1. [**Prepare your development environment**](#prepare-your-development-environment)
+> #### 1.2.  [**First contribution**](#first-contribution)
+> #### 1.3. [**Add new feature, bug fixing or improvement**](#add-new-feature-bugfixing-or-improvement)
+> #### 1.4.  [**Pull Request's approval**](#pull-request-approval)
+> #### 1.5.   [**After your pull request's approval**](#after-your-pull-request-approval)
```
Comment (wiliansilvazup) [inline CONTRIBUTING.md:18]:
```suggestion
> #### 3.1. [**Prepare your development environment**](#prepare-your-development-environment)
> #### 3.2.  [**First contribution**](#first-contribution)
> #### 3.3. [**Add new feature, bug fixing or improvement**](#add-new-feature-bugfixing-or-improvement)
> #### 3.4.  [**Pull Request's approval**](#pull-request-approval)
> #### 3.5.   [**After your pull request's approval**](#after-your-pull-request-approval)
```
the correct would be in this format no?

File: README.md [hunk 20]
```diff
@@ -26,29 +26,31 @@
...
-        5. [**Using Visual Studio Code**](#Using-Visual-Studio-Code)
-        6. [**Using the Pipeline**](#Using-the-Pipeline)
-4. [**Contributing**](#Contributing)
-5. [**Roadmap**](#Community)
-6. [**License**](#License)
-7. [**Community**](#Community)
+### 1. [**About**](#about)
+### 2. [**Getting started**](#getting-started)
+>#### 1.1.   [**Requirements**](#requirements)
+>#### 1.2.  [**Installation**](#installing-horusec)
```
Comment (wiliansilvazup) [inline README.md:32]:
```suggestion
>#### 2.1.   [**Requirements**](#requirements)
>#### 2.2.  [**Installation**](#installing-horusec)
```

the correct would be in this format no?

**Review by wiliansilvazup — CHANGES_REQUESTED** [review]
Wait double check this changes

File: README.md (hunk 20 above)
Comment (jessica-tw) [reply README.md:32]: it is, fixing now! ;)

**Review by wiliansilvazup — APPROVED** [review]

**Pull request merged**